packages/core/src/prompts/lesson-assistant/index.ts

```diff
@@ -68,7 +69,7 @@ export const getPromptParts = (props: TemplateProps): TemplatePart[] => {
       : undefined,
     americanToBritishSection,
     languageAndVoice,
-    schema,
+    props.isUsingStructuredOutput ? undefined : schema,
```

Comment:
No need for schema in prompt and response_schema. Should shave off a few tokens.

Reply:
Great - glad we've worked that out